.github/workflows/release.yml

```diff
@@ -48,8 +48,6 @@ jobs:
         run: python -u ./dev/releases/make_archives.py
       - name: "Make GitHub release"
         if: ${{ startsWith(github.ref, 'refs/tags/v') }}
-        # TODO: we should check whether the tag triggering the release
-        # has the right name
         run: python -u ./dev/releases/make_github_release.py
```

Comment:
Shouldn't the tag name be passed as an argument here?

Reply:
Done.